client/tailscale/localclient.go

```diff
@@ -1505,3 +1505,16 @@ func (w *IPNBusWatcher) Next() (ipn.Notify, error) {
 	}
 	return n, nil
 }
+
+// SuggestExitNode returns the tailcfg.StableNodeID of a suggested exit node to connect to.
```

Comment:
"...returns the tailcfg.StableNodeID, name and tailcfg.Location of a suggested..."